Model/Config.php

```diff
@@ -188,7 +189,7 @@ public function getCmsBlocks()
      */
     private function initialize()
     {
-        $data = $this->reader->read();
+        $data               = $this->reader->read();
```

Comment:
hmm, I didn't intend to change this file, maybe I got code style wrong, will cleanup if necessary

Reply:
I do not care too much, fine for me :)